package.json

```diff
@@ -17,6 +17,7 @@
     "dependencies": {
         "@eyeseetea/d2-api": "^1.11.0",
         "cmd-ts": "0.13.0",
+        "csv-writer": "^1.6.0",
```

Comment:
can you change the name and description? License should be GPL. The author email should be info@eyeseetea.com. If any of this applies to the skeleton feel free to add a review to @eperedo 's skeleton work